Add default organization community health files

d33bs opened the pull request:
This PR adds the default community health files for all repositories within the Software Gardening organization. I've deferred to omit `CONTRIBUTING.md` for now to defer to repo-specific contribution guidelines. I feel that `GOVERNANCE.md` may need to be added at a later point as related aspects emerge.

Thanks for any thoughts and feedback you may have!

References #2 

Note: I feel completion of #2 requires funding attribution details based on the BSSw Fellowship this work relates to (which I feel should be included by default throughout). I've reached out about this and hope to follow up with a subsequent PR at a later point.

d33bs committed:
add default security documentation

d33bs committed:
adjust the readme for better context

d33bs committed:
add code of conduct

d33bs committed:
add license file extension

d33bs committed:
adjust copyright

d33bs committed:
add attributions

gwaybio commented on SECURITY.md:
```diff
@@ -0,0 +1,9 @@
+# Reporting Security Issues
+
+Software Gardening maintainers and community take security concerns seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
```
```suggestion
Software Gardening maintainers and community take security concerns seriously.
We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
```

d33bs committed:
Update SECURITY.md

Co-authored-by: Gregory Way <gregory.way@gmail.com>

vincerubinetti reviewed (approved):
This seems very well written and covers many bases. The only thing that stuck out to me is maybe the difference between the "correction" and "warning" levels isn't clear enough? Also perhaps "action" or "problem" or "violation" is a simpler term than "community impact"? I was slightly confused with that at first glance. "violation -> consequence" would be a pretty clear pairing I think, and matches the terminology at the top of the section.

I have no experience writing this type of documentation so take this with a grain of salt.

d33bs committed:
simplify CoC language

Co-Authored-By: Vincent Rubinetti <8326331+vincerubinetti@users.noreply.github.com>

d33bs commented:
Thanks @gwaybio and @vincerubinetti for your reviews!

> The only thing that stuck out to me is maybe the difference between the "correction" and "warning" levels isn't clear enough? Also perhaps "action" or "problem" or "violation" is a simpler term than "community impact"? I was slightly confused with that at first glance. "violation -> consequence" would be a pretty clear pairing I think, and matches the terminology at the top of the section.

I moved from "Correction" to "First Warning" and "Second Warning". The (originally titled) "Correction" step includes a consequence which mentions the word warning. I understood an implied "3 strikes" sort of cadence (with the 3rd strike here being a ban of some kind), so tried to make this more firm / understandable. I also moved to "Action(s)" -> "Consequence" structure to simplify based on your suggestions.
